readalongs/cli.py

```diff
@@ -77,12 +83,58 @@ def get_click_file_name(click_file):
 
 
 @click.version_option(version=__version__, prog_name="readalongs")
-@click.group(cls=FlaskGroup, create_app=create_app, context_settings=CONTEXT_SETTINGS)
+@click.group(context_settings=CONTEXT_SETTINGS)
 def cli():
-    """Management script for Read Along Studio."""
+    """Management script for ReadAlong Studio.
```

Comment:
Nice, that is so much better a global help message. All of it useful. It formats nicely on screen, did you check that it generates a good output in readthedocs? Actually, it doesn't matter, our readthedocs does not embed the output of `readalongs -h`, it gives this information in more details instead.